tests/test_utils.py

```diff
@@ -1,31 +1,36 @@
+# Builtin libs
 import os
-from collections import OrderedDict, namedtuple
+import re
 import glob
-
 import traceback
+import warnings
+from collections import OrderedDict, namedtuple
+import multiprocessing as mp
 
+# Computation libs
 import numpy as np
 import pandas as pd
 import xarray as xr
-import glob
-import re
+
+# Plottling libs
 import matplotlib
-matplotlib.use('Agg')
```

Comment:
Hmm, I think I added this because I failed plotting when I ran the science test on hydra nodes. But I don't really understand this so if this is indeed not necessary then ignore my comment.